packages/at_onboarding_cli/lib/src/onboard/at_onboarding_service_impl.dart

```diff
@@ -219,23 +223,13 @@ class AtOnboardingServiceImpl implements AtOnboardingService {
         atKeysMap[AuthKeyType.encryptionPrivateKey]!,
         atKeysMap[AuthKeyType.selfEncryptionKey]!);
 
-    if (atOnboardingPreference.downloadPath != null) {
-      //construct download path to match standard atKeys file name convention
-      if (!(atOnboardingPreference.downloadPath!.endsWith('.atKeys'))) {
-        atOnboardingPreference.downloadPath = path.join(
-            atOnboardingPreference.downloadPath!, '${_atSign}_key.atKeys');
-      }
-    } else if (atOnboardingPreference.atKeysFilePath != null) {
-      //if atKeysFilePath points to a directory and not a file, create a file in the provided directory
-      if (!(atOnboardingPreference.atKeysFilePath!.endsWith('.atKeys'))) {
-        atOnboardingPreference.atKeysFilePath = path.join(
-            atOnboardingPreference.atKeysFilePath!, '${_atSign}_key.atKeys');
-      }
+    if (!atOnboardingPreference.atKeysFilePath!.endsWith('.atKeys')) {
```

Comment:
atOnboardingPreference.atKeysFilePath ??= HomeDirectoryUtil.getAtKeysPath(_atSign);
  This call appends .atKeys at the end..is this condition required?

Reply:
If someone provides an atKeyFile path as /home/Desktop/@abcd, we need to ensure that the file name is @abcd.atKeys. This condition ensures that the filename is whatever the user provided but the format is .atKeys.